.github/workflows/test-wordpress-version.yml

```diff
@@ -0,0 +1,61 @@
+name: 'Test WordPress Version'
+
+on:
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  wp4721:
+    runs-on: ubuntu-latest
+    name: Expect 4.7.21
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Start wp-env
+        uses: ./
+        with:
+          core: "WordPress/WordPress#4.7.21"
+
+      - name: Expect WordPress version 4.7.21
+        run: |
+          wp-env run cli "wp core version"
+          if [[ $(wp-env run cli "wp core version") != "4.7.21" ]]; then exit 1; fi
+
+  wp54:
+    runs-on: ubuntu-latest
+    name: Expect 5.4
```

Comment:
And 5.4 ? What is the logic behind the choosing of these versions?

Reply:
Same answer, regarding testing the Action config.